tools/enable-kdump.sh

```diff
@@ -30,6 +30,35 @@ supported_version=( "12-SP2"
     "15-SP2"
 )
 
+# get OS name and OS version
+# /etc/os-release file has this information
+# in the form of key value pair so these can be
+# imported in shell varible
+eval $(cat /etc/os-release | sed -e s"@: @=@")
+
+# check if the os and version is supported by this script
+supported="false"
+for i in "${supported_os[@]}"; do
```

Comment:
Minor: Can we do something more efficient to avoid multiple for loop iteration?

Reply:
This will be updated in next iteration of update.